services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/client/CloudDccFeignHttpClientProvider.java

```diff
@@ -53,39 +53,30 @@ public CloudDccFeignHttpClientProvider(DistributionServiceConfig config) {
   @Bean
   public Client createFeignClient() {
     return new ApacheHttpClient(
-        federationHttpClientFactory().createBuilder().build());
+        dccHttpClientFactory().createBuilder().build());
   }
 
   /**
    * Creates an {@link ApacheHttpClientFactory} that with no SSL certificates and no host names.
    */
   @Bean
-  private ApacheHttpClientFactory federationHttpClientFactory() {
+  @Profile("dcc-client-factory")
```

Comment:
what's this new profile about?
when is it active?
PRD is running normally only with `cloud` profile active!!!

Reply:
Hi Hilmar. This bean is only used for testing purposes. The tests are failing without this being a bean.